audible.py

```diff
@@ -287,8 +288,13 @@ def get_albums(self, query):
             if len(products_without_unreleased_entries) < len(products):
                 # see https://github.com/laxamentumtech/audnexus/issues/239
                 self._log.info(f"Excluded {len(products) - len(products_without_unreleased_entries)} books which have not been released from consideration.")
-            
-            return [self.get_album_info(p["asin"]) for p in products_without_unreleased_entries]
+            out = []
+            for p in products_without_unreleased_entries:
+                try:
+                    out.append(self.get_album_info(p['asin']))
+                except urllib.error.HTTPError:
+                    pass
```

Comment:
Could you log an error so that there is some indication that this is happening? For example, copying the handling done a few lines below.

Reply:
Done